3p/frame.max.html

```diff
@@ -2,7 +2,7 @@
 <head>
 <meta charset="utf-8">
 <meta name="robots" content="noindex">
-<script src="./integration.js"></script>
+<script src="./f.js"></script>
```

Comment:
You will need to update the gulpfile.  

See https://github.com/ampproject/amphtml/blob/master/gulpfile.js#L366

It is currently looking for integration.js. 

Would it be possible to just remove this build step by constructing this src with AMP_CONFIG?